test/key/processMIME.spec.ts

```diff
@@ -13,46 +12,51 @@ import {
     key
 } from './processMIME.data';
 
-test('it can process multipart/signed mime messages and verify the signature', async (t) => {
-    const { body, verified } = await processMIME(
-        {
-            publicKeys: await getKeys(key)
-        },
-        multipartSignedMessage
-    );
-    t.is(verified, VERIFICATION_STATUS.SIGNED_AND_VALID);
-    t.is(body, multipartSignedMessageBody);
-});
+describe('processMIME', () => {
+    it('it can process multipart/signed mime messages and verify the signature', async () => {
+        const { body, verified } = await processMIME(
+            {
+                publicKeys: await getKeys(key)
+            },
+            multipartSignedMessage
+        );
+        expect(verified).to.equal(VERIFICATION_STATUS.SIGNED_AND_VALID);
+        expect(body).to.equal(multipartSignedMessageBody);
+    });
 
-test('it can process multipart/signed mime messages and verify the signature with extra parts at the end', async (t) => {
-    const { body, verified } = await processMIME(
-        {
-            publicKeys: await getKeys(key)
-        },
-        extraMultipartSignedMessage
-    );
-    t.is(verified, VERIFICATION_STATUS.SIGNED_AND_VALID);
-    t.is(body, 'hello');
-});
+    it('it can process multipart/signed mime messages and verify the signature with extra parts at the end', async () => {
+        const { body, verified } = await processMIME(
+            {
+                publicKeys: await getKeys(key)
+            },
+            extraMultipartSignedMessage
+        );
+        expect(verified).to.equal(VERIFICATION_STATUS.SIGNED_AND_VALID);
+        expect(body).to.equal('hello');
+    });
+    
+    it('it does not verify invalid messages', async () => {
+        const { verified, body } = await processMIME(
+            {
+                publicKeys: await getKeys(key)
+            },
+            invalidMultipartSignedMessage
+        );
+        expect(verified).to.equal(VERIFICATION_STATUS.NOT_SIGNED);
+        expect(body).to.equal('message with missing signature');
+    });
+    
+    it('it can parse messages with special characters in the boundary', async () => {
+        const { verified, body } = await processMIME(
+            {
+                publicKeys: await getKeys(key)
+            },
+            multiPartMessageWithSpecialCharacter
+        );
+        expect(verified).to.equal(VERIFICATION_STATUS.SIGNED_AND_VALID);
+        expect(body).to.equal('hello');
+    });
+    
+})
```

Comment:
This is very nitpicky but
```suggestion
});
```
(A few other files are missing the last semicolon too)

Reply:
I'll try to have the linter enforce this + the EOF new line